com.unity.shadergraph/Editor/Data/Nodes/Mesh Deformation/LinearBlendSkinningNode.cs

```diff
@@ -13,15 +13,13 @@ class LinearBlendSkinningNode : AbstractMaterialNode, IGeneratesBodyCode, IGener
         public const int kPositionOutputSlotId = 3;
         public const int kNormalOutputSlotId = 4;
         public const int kTangentOutputSlotId = 5;
-        public const int kSkinMatricesOffsetSlotId = 6;
 
         public const string kSlotPositionName = "Position";
         public const string kSlotNormalName = "Normal";
         public const string kSlotTangentName = "Tangent";
         public const string kOutputSlotPositionName = "Position";
         public const string kOutputSlotNormalName = "Normal";
         public const string kOutputSlotTangentName = "Tangent";
```

Comment:
Is it preferred to have 'InPosition' and 'OutPosition', instead of 'Position' for both input and output port? What is the SG convention?

Reply:
I would say rather than in/out, I would prefer something a little more descriptive like "Vertex Position" and "Skinned Position" . 
It will make the node render pretty wide though, but something that more clearly described what the input and output actually are is preferable.